File: Excel_Programs/Java/question9.java
class question9 
{
    public static void main(String[] args) 
    {
        int arr[] = {6, 12, 9, 13, 17};
        int n=arr.length;
        int min=arr[0];
        int max=0;
        for(int i=0;i<n;i++)
        {
            if(min>arr[i])
            {
                min= arr[i];
            }
            if(max<arr[i])
            {
                max= arr[i];
            }
        }
        System.out.println(max-min);
    }    
}

File: Excel_Programs/Python/question9.py
arr = [6, 12, 9, 13, 17]
max=0
min=arr[0]
for i in range(len(arr)):
    if(min>arr[i]):
        min=arr[i]
    if(max<arr[i]):
        max=arr[i]
print(max-min)